### PR TITLE
Update toggl-track from 7.5.441 to 7.5.461

### PR DESCRIPTION
**Pull request by tyrann0us**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by tyrann0us** [comment]
Sorry, accidentally closed the PR. This is my first contribution, so please bear with me.
No idea why the check failed with this:
```
Error: 1 problem in 1 cask detected
audit for toggl-track: failed
 - Version '7.5.461' differs from '7.5.441' retrieved by livecheck.
 - Version '7.5.461' differs from '7.5.441' retrieved by livecheck.
Error: Version '7.5.461' differs from '7.5.441' retrieved by livecheck.
Error: Process completed with exit code 1.
```

**Comment by bevanjkay** [comment]
@tyrann0us The link from the Toggl Track website is still pointing to the previous version - https://support.toggl.com/en/articles/6151856-toggl-track-desktop-app-for-macos - see download link here. So I would say it is likely that `7.5.441` is the correct current version.

File: Casks/toggl-track.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@
   homepage "https://www.toggl.com/track/toggl-desktop/"
 
   livecheck do
-    url "https://toggl-open-source.github.io/toggldesktop/assets/releases/darwin_stable_appcast.xml"
-    strategy :sparkle
+    url "https://github.com/toggl-open-source/toggldesktop"
+    strategy :github_latest
```
Comment (SMillerDev) [inline Casks/toggl-track.rb:13]: This actually makes it worse:

> Version '7.5.461' differs from '7.5.363' retrieved by livecheck.

Comment (tyrann0us) [reply Casks/toggl-track.rb:13]: Yes, I noticed this as well. See the linked issue (https://github.com/toggl-open-source/toggldesktop/issues/4906); maybe they'll consider creating stable releases again.

**Review by tyrann0us — COMMENTED** [review]
@bevanjkay
> So I would say it is likely that 7.5.441 is the correct current version.

I'm not so sure about this TBH, because according to the GitHub releases, 7.5.363 is the latest stable version (see @SMillerDev's line comment) and all newer releases are marked as "Pre-Release". This makes me believe that Toggl simply isn't too caring about their version management for the 7.* branch now that the latest version is 8.* (which is only distributed through the Mac App Store, unfortunately).

**Comment by p-linnane** [comment]
The repo, website, and appcast all offer `7.5.441` as stable. Additionally, there is a message on the repo's README.md explaining the app is in maintenance mode:

>⚠️ Project Status: maintenance mode (bug fixes only).
>
>At the moment, we will not implement nor accept pull requests implementing new features in this repository.
>
>We’re currently reworking the internals of our macOS and Windows desktop apps, and we plan to release the refreshed apps soon. If you’re interested in trying the app now, take the beta version for a spin. For more information, please visit the [macOS](https://github.com/toggl/track-macos-feedback) or [Windows](https://github.com/toggl/track-windows-feedback) feedback repository.

Given all the above, I am going to close this out until a new stable version is offered at the above listed locations.

**Comment by tyrann0us** [comment]
> until a new stable version is offered at the above listed locations.

From Toggl support:
> We discussed this with our team and v7.5.461 is the latest dev build for the desktop app. The latest stable release is still v7.5.441.

So yes, no cask update for the time being.

**Pull request closed without merging**